extensions/terminal-suggest/src/fig/autocomplete-parser/constants.ts

```diff
@@ -0,0 +1,33 @@
+const AWS_SPECS = ["aws", "q"];
+const UNIX_SPECS = [
+  "cd",
+  "git",
+  "rm",
+  "ls",
+  "cat",
+  "mv",
+  "ssh",
+  "cp",
+  "chmod",
+  "source",
+  "curl",
+  "make",
+  "mkdir",
+  "man",
+  "ln",
+  "grep",
+  "kill",
+];
+const EDITOR_SPECS = ["code", "nano", "vi", "vim", "nvim"];
+const JS_SPECS = ["node", "npm", "npx", "yarn"];
+const MACOS_SPECS = ["brew", "open"];
+const OTHER_SPECS = ["docker", "python"];
+
+export const MOST_USED_SPECS = [
+  ...AWS_SPECS,
+  ...UNIX_SPECS,
+  ...EDITOR_SPECS,
+  ...JS_SPECS,
+  ...MACOS_SPECS,
+  ...OTHER_SPECS,
+];
```

Comment:
This might feed into their sorting?

Reply:
Just for loading: https://github.com/aws/amazon-q-developer-cli/blob/f66e0b0e917ab185eef528dc36eca56b78ca8b5d/packages/autocomplete-parser/src/loadHelpers.ts#L235-L246